packages/block-editor/src/components/inserter/menu.js

```diff
@@ -149,7 +149,18 @@ function InserterMenu(
 	const showZoomOut =
 		showPatternPanel && !! window.__experimentalEnableZoomedOutPatternsTab;
 
-	useZoomOut( showZoomOut );
+	const { __unstableSetEditorMode } = useDispatch( blockEditorStore );
+	useEffect( () => {
+		if ( showZoomOut ) {
+			__unstableSetEditorMode( 'zoom-out' );
+		}
+
+		return () => {
+			if ( showZoomOut ) {
+				__unstableSetEditorMode( 'edit' ); // TODO: set back to whatever it was previously
+			}
+		};
+	}, [ showZoomOut ] );
```

Comment:
Why did you have to switch away from `useZoomOut` here and below https://github.com/WordPress/gutenberg/pull/63870/files#diff-fe01ab6018c513cc3b7cdb7e27a70b4aad3d73c1fe8d1267a91c1d80b6a4334fR61-R75?

Reply:
useZoomOut works when the component mounts, whereas this is something that needs to continue to react to changes in the component.

Reply:
If we leave this in then changing the value from 50% to 100% in the preview dropdown doesn't work, as this component forces it back to 50%.

Reply:
Actually I found a way to use the hook, as long as we change the dependency array.